plugins/org.jkiss.dbeaver.ui/src/org/jkiss/dbeaver/ui/dialogs/EditTextDialog.java

```diff
@@ -105,8 +120,9 @@ protected void okPressed() {
         super.okPressed();
     }
 
-    public static String editText(Shell parentShell, String title, String text) {
+    public static String editText(Shell parentShell, String title, String text, String infoLabelText) {
```

Comment:
Please provide an overload with an additional parameter instead of modifying the existing method. This change may break existing code that relies on it outside of the `dbeaver/dbeaver` repo.